tests/functional/modules/test_zos_operator_func.py

```diff
@@ -119,13 +120,31 @@ def test_zos_operator_positive_verbose_with_quick_delay(ansible_zos_module):
         assert result.get('elapsed') <= (2 * wait_time_s)
 
 
+def test_zos_operator_positive_verbose_blocking(ansible_zos_module):
+    if zoau_version_checker.is_zoau_version_higher_than("1.2.4.5"):
+        hosts = ansible_zos_module
+        wait_time_s=5
+        results = hosts.all.zos_operator(
+            cmd="d u,all", verbose=True, wait_time_s=wait_time_s
+        )
+
+        for result in results.contacted.values():
+            assert result["rc"] == 0
+            assert result.get("changed") is True
+            assert result.get("content") is not None
+            # Account for slower network
+            assert result.get('elapsed') >= wait_time_s
+
+
+
 def test_response_come_back_complete(ansible_zos_module):
     hosts = ansible_zos_module
-    results = hosts.all.zos_operator(cmd="\$dspl")
+    results = hosts.all.zos_operator(cmd="\\$dspl")
```

Comment:
Why is this backslash added?

Reply:
With 1 backslash, it was being flagged by a linter (don't remember which one).

Reply:
I see, I ran the same command in my local and works fine, was worried it would change something 👍